src/components/SingleDatePicker.jsx

```diff
@@ -218,6 +218,12 @@ export default class SingleDatePicker extends React.Component {
       values(visibleDays).forEach((days) => {
         Object.keys(days).forEach((day) => {
           const momentObj = moment(day);
+          if (isOutsideRange(momentObj)) {
```

Comment:
Could you elaborate for me @timhwang21? There are a couple other checks (`isDayBlocked()` and `isDayHighlighted()` that happen right afterwards, and I believe `this.isBlocked()` checks both `isDayBlocked` and `isOutsideRange` (on [line 628](https://github.com/jlyman/react-dates/blob/master/src/components/SingleDatePicker.jsx#L628)).